pkg/store/postgres.go

```diff
@@ -36,6 +36,7 @@ var (
 				branch TEXT,
 				sha TEXT,
 				identifier BIGINT,
+				actor TEXT,
```

Comment:
This is a migration list, please add an element to the list with an alter table statement, otherwise this column won't be available for existing installations.